tests/test_autonomy/test_cli/test_deploy/test_from_token.py

```diff
@@ -62,6 +67,9 @@ def test_from_token(
         ), mock.patch(
             "autonomy.cli.fetch.get_ipfs_node_multiaddr",
             new=lambda: "/dns/registry.autonolas.tech/tcp/443/https",
+        ), mock.patch(
+            "autonomy.deploy.chain.ServiceRegistry._resolve_from_ipfs",
+            return_value=mock_ipfs_response,
```

Comment:
we mock the ipfs http request. I don't see a point why this should be using a real gateway, just introduces an external dependency.